src/main/java/me/juancarloscp52/entropy/events/db/LowFPSEvent.java

```diff
@@ -41,12 +42,7 @@ public void endClient() {
     }
 
     @Override
-    public String type() {
-        return "fps";
-    }
```

Comment:
This disappeared. On purpose?

Reply:
Yes, this was the only event that had this type.